### PR TITLE
BrenKt-TRANS-SK-1331-WEB-EmailPro-configuring_Gmail_Android 

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

**Comment by BrenKt** [comment]
Language check to do please

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pl-pl.md [hunk 1]
```diff
@@ -1,100 +1,174 @@
----
-title: 'Konfiguracja konta E-mail Pro na urządzeniu z Android przy użyciu aplikacji Gmail'
-excerpt: 'Dowiedz się, jak skonfigurować konto E-mail Pro na urządzeniu z systemem Android przy użyciu aplikacji Gmail'
-updated: 2020-03-18
----
+--- 
+title: "E-mail Pro - Konfiguracja konta e-mail w programie Gmail na urządzeniu z systemem Android"
+excerpt: "Dowiedz się, jak skonfigurować konto E-mail Pro na urządzeniu z systemem Android poprzez aplikację Gmail"
+Updated: 2024-03-15
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pl-pl.md]:
```suggestion
updated: 2024-03-15
```

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pl-pl.md [hunk 2]
```diff
@@ -1,100 +1,174 @@
...
+>> 1. Przejdź do menu w lewym górnym rogu ekranu<br><br>
+>> 2. Wybierz `Parametry`{.action}<br><br>
+>> 3. Wybierz `Dodaj konto`{.action}<br><br>![emailpro](images/android-existing.png){thumbnail .h-600}
+
+Postępuj zgodnie z kolejnymi instrukcjami, przechodząc przez poniższe karty:
+
+> [!tabs]
+> **Etap 1**
+>> Z menu Typy kont e-mail wybierz `Inny`{.action}.<br><br>
+>> ![emailpro](obrazy/emailpro-android-01.png){.thumbnail .h-600}
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pl-pl.md]:
```suggestion
>> ![emailpro](images/emailpro-android-01.png){.thumbnail .h-600}
```

Same translation mistake is repeated below several times

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pl-pl.md [hunk 3]
```diff
@@ -1,100 +1,174 @@
...
+
+> [!success]
+>
+> OVHcloud oferuje aplikację internetową dostępną pod adresem <https://www.ovhcloud.com/pl/mail/> umożliwiającą korzystanie z Twojego konta e-mail przy użyciu wyszukiwarki. Możesz się do niej zalogować, używając tych samych danych, których używasz do logowania się do konta e-mail.
 
-- **jeśli nie jest skonfigurowane żadne konto**: postępuj zgodnie z instrukcjami etapu powitalnego, następnie kliknij `Dodaj konto e-mail`{.action}. Wybierz `Inne`{.action}; 
+### Parametry POP, IMAP i SMTP
 
-- **jeśli inne konto jest już skonfigurowane**: kliknij piktogram w postaci trzech myślników na górze po lewej stronie, a następnie piktogram w postaci strzałki po prawej stronie nazwy skonfigurowanego konta. Następnie kliknij `Dodaj konto`{.action} i wybierz `Inne`{.action}. 
+Jeśli chcesz otrzymywać e-maile, wybierz rodzaj konta. Zalecamy użycie **IMAP**. Możesz jednak wybrać **POP**. Aby dowiedzieć się, jak to działa, sprawdź naszą sekcję ["POP lub IMAP, jaka jest różnica?"] (#popimap)
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pl-pl.md]:
```suggestion
Jeśli chcesz otrzymywać e-maile, wybierz rodzaj konta. Zalecamy użycie **IMAP**. Możesz jednak wybrać **POP**. Aby dowiedzieć się, jak to działa, sprawdź naszą sekcję ["POP lub IMAP, jaka jest różnica?"](#popimap)
```

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pt-pt.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+Os endereços de e-mail do serviço E-mail Pro podem ser configurados em vários softwares de e-mail compatíveis. Isto permite-lhe enviar e receber e-mails no dispositivo que preferir. Encontre neste guia as etapas de configuração de um endereço de e-mail E-mail Pro a partir da aplicação Gmail presente nos dispositivos Android.
+
+**Saiba como configurar uma conta E-mail Pro no Android através da aplicação Gmail.**
+
+> [!warning]
+>
+> A OVHcloud oferece-lhe serviços cuja configuração, gestão e responsabilidade é da sua responsabilidade. Assim, deverá assegurar o seu bom funcionamento.
+>
+> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [fornecedor especializado](https://partner.ovhcloud.com/pt/) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte a secção « Ir mais longe » deste guia.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pt-pt.md]:
```suggestion
> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [fornecedor especializado](/links/partner) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte a secção « Ir mais longe » deste guia.
```

Same link can be replaced on other translations

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pt-pt.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+
+## Requisitos
+
+- Ter um plano [E-mail Pro](https://www.ovhcloud.com/pt/emails/email-pro/).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pt-pt.md]:
```suggestion
- Ter um plano [E-mail Pro](/links/email-pro).
```

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pt-pt.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+
+Depois de configurar o endereço de e-mail, já só precisa de o utilizar ! Pode desde já enviar e receber mensagens a partir da sua aplicação Gmail.
+
+> [!success]
+>
+> A OVHcloud disponibiliza uma aplicação web que pode utilizar para aceder ao seu e-mail diretamente a partir do browser da Web: <https://www.ovhcloud.com/pt/mail/>. Para aceder, só precisa dos dados de acesso do seu endereço de e-mail.
+
+### Definições POP, IMAP e SMTP
+
+Para a receção dos e-mails, ao escolher o tipo de conta, recomendamos uma utilização em **IMAP**. No entanto, pode selecionar **POP**. Para compreender o funcionamento respetivo, consulte a nossa secção [« POP ou IMAP, qual é a diferença? »](#popimap)
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pt-pt.md]:
```suggestion
Para a receção dos e-mails, ao escolher o tipo de conta, recomendamos uma utilização em **IMAP**. No entanto, pode selecionar **POP**. Para compreender o funcionamento respetivo, consulte a nossa secção [POP ou IMAP, qual é a diferença?](#popimap)
```

**Review by OVH-Team-Guides — COMMENTED** [review]
Systran warnings are missing

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**